README.rst

```diff
@@ -57,23 +57,68 @@ our homepage_.
 .. |weblate| image:: https://weblate.bubu1.eu/widgets/searxng/-/searxng/svg-badge.svg
    :target: https://weblate.bubu1.eu/projects/searxng/
 
+Contact
+"""""""
+
+Matrix: `#searxng:matrix.org <https://matrix.to/#/#searxng:matrix.org>`_
+
+IRC:  `#searxng on libera.chat <https://web.libera.chat/?channel=#searxng>`_ which is bridged to Matrix.
 
-SearXNG is a fork of `searx`_:
 
-* `the differences with searx <https://github.com/searxng/searxng/issues/46>`_
-* `exhaustive changelog <https://github.com/searxng/searxng/wiki/Changes-from-version-1.0.0>`_
+Differences with searx
+""""""""""""""""""""""
+
+SearXNG is a fork of `searx`_.
+Here some of the changes:
 
 .. _searx: https://github.com/searx/searx
 
 
-Contact
-"""""""
+User experience
+~~~~~~~~~~~~~~~
 
-Matrix: `#searxng:matrix.org <https://matrix.to/#/#searxng:matrix.org>`_
+-  Huge update of the simple theme:
 
-IRC:  `#searxng on libera.chat <https://web.libera.chat/?channel=#searxng>`_ which is bridged to Matrix.
+   -  usable on desktop, tablet and mobile.
+   -  light and dark versions (you can choose in the preferences)
+   -  on the way to support RTL languages.
+
+-  the translations are up to date, you can contribute `here`_
```

Comment:
I prefer descriptive link labels, e.g. you can contribute on [Weblate](https://weblate.bubu1.eu/projects/searxng/searxng/)

Reply:
Fixed